examples/src/main/java/alluxio/examples/AlluxioFS.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.examples;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileInStream;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.FileSystem;
+
+/**
+ * Example program to use Alluxio native API to read and write files.
+ *
+ */
+public class AlluxioFS
+{
+  /**
+   * Entry point for the {@link AlluxioFS} program.
+   *
+   * @param args command-line arguments
+   */
+  public static void main(String[] args) throws
+      alluxio.exception.FileAlreadyExistsException,
+      alluxio.exception.InvalidPathException,
+      alluxio.exception.AlluxioException,
+      java.io.IOException
+  {
+    System.out.println("Start Alluxio Native FS write/read");
+    FileSystem fs = FileSystem.Factory.get();
+    AlluxioURI path = new AlluxioURI("/AlluxioFS.txt");
+
+    FileOutStream out = fs.createFile(path);
+    byte[] buffout = new byte[4096];
```

Comment:
replace the magic number with a const variable?

Reply:
Fixed.